pkg/armhelpers/azurestack/httpMockClientData/deployVMRequest.json

```
@@ -1831,13 +1831,13 @@
                 "value": "latest"
             },
             "servicePrincipalClientId": {
-                "value": "85115f84-ef7b-4ddb-b44d-b3a9d3b1990d"
+                "value": "servicePrincipalClientId"
```

Do you think we could make this value different from the key so it's not confusing?

```suggestion
                "value": "fakeClientId"
```